typescript/floating_promises.md

```diff
@@ -0,0 +1,44 @@
+# Floating Promises
```

Comment:
What do you think about making an `examples` directory in `typescript` to house this and others?